### PR TITLE
Add quote to yaml reserved chars in fixtures.

### PR DESCRIPTION
**Pull request by NiR-**

### CODE REVIEW

**Comment by Seldaek** [comment]
Would you mind updating the markdown files of the docs too? We shouldn't promote stuff that won't work..

**Comment by NiR-** [comment]
@Seldaek done ;)

**Comment by Soullivaneuh** [comment]
@NiR- since when using `@thing` directly cause problems with yaml? Don't have this issue on my projects.

**Comment by jenkoian** [comment]
@Soullivaneuh it's to work with Symfony 3 due to https://github.com/symfony/symfony/pull/16285

**Comment by Soullivaneuh** [comment]
Oh I see, thanks for the tip. :+1:

File: doc/relations-handling.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ Nelmio\Entity\User:
 Nelmio\Entity\Group:
     group1:
         name: Admins
-        owner: @user1
+        owner: '@user1'
 ```
 
-Alice also allows you to directly reference objects' properties using the ```@name->property``` notation.
+Alice also allows you to directly reference objects' properties using the ```'@name->property'``` notation.
```
Comment (Seldaek) [inline doc/relations-handling.md]: Sorry but those at line 7, 35 and here should not be quoted IMO, as they are not in the Yaml. The quotes aren't part of the alice syntax.

**Comment by Einenlum** [comment]
Up ?

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**